src/views/transactions/transactionDetails/AccountMetadataView.ts

```diff
@@ -0,0 +1,13 @@
+import {AccountMetadataTransaction} from 'nem2-sdk';
```

Comment:
I propose to move ``transactions/transactionDetails`` to ``transaction/details``. Also, each view could be named as ``tx_name.view.ts`` to follow the naming format of services, commands, resolvers...

Reply:
I think we should fix a naming format now, because it is not really clear:
Following what we have now, I think we should stick to `camelCase.filecategory.ts`

```
src/commands/converter/hexToBase32Address.ts
src/resolvers/restrictionAccount.resolver.ts
test/options-resolver.spec.ts
```

Reply:
however, we should take a decision for folder names

I personally prefer `folder-name` to `folderName `

Reply:
the thing I'm not confortable with are the files that are not a view, service, repository... _per se_,
for example the TableBuilder class. I don't know how to name them

Reply:
> src/commands/converter/hexToBase32Address.ts

commands cannot be renamed as `camelCase.filecategory.ts` because of the framework takes the name of the file as the name of the command. 

> I personally prefer folder-name to folderName

Ok.

> the thing I'm not comfortable with is the files that are not a view, service, repository... per se,

TableBuilder could be considered a model named ``Table``?

Reply:
I moved it to model but kept the TableBuilder name since there is an external import named Table already